autogpt_platform/backend/schema.prisma

```diff
@@ -257,9 +428,18 @@ model AnalyticsMetrics {
   user   User   @relation(fields: [userId], references: [id], onDelete: Cascade)
 }
 
+//////////////////////////////////////////////////////////// 
+//////////////////////////////////////////////////////////// 
+////////   ACCOUNTING AND CREDIT SYSTEM TABLES    //////////
+//////////////////////////////////////////////////////////// 
+//////////////////////////////////////////////////////////// 
+
 enum UserBlockCreditType {
   TOP_UP
   USAGE
+  COMMISSION
+  PURCHASE
+  SALE
 }
 
 model UserBlockCredit {
```

Comment:
can this be renamed to be more clear? 
- `UserBlockCreditMutation`
- `UserBlockCreditTransaction`
- `UserCreditMutation`
- `UserCreditTransaction`

Reply:
I'm waiting for @majdyz to weight in on this part before making more changes

Reply:
We definitely could! Is the scope of this PR DB renamings though? Can we simply start with the DB changes while retaining the name for now? Because it's going to be complicated changes already. You also need to propagate the code changes. 

I don't think others would be able to review it properly if the changes are humongous.